src/pages/registration/contributors_tab/components/Contributors.tsx

```diff
@@ -21,30 +25,29 @@ export const ContributorEditors = () => {
   } = useFormikContext<Registration>();
   const contributorsError = errors.entityDescription?.contributors;
   const contributorsTouched = touched.entityDescription?.contributors;
-  const editorsRef = useRef(contributors);
+  const contributorsRef = useRef(contributors);
 
   useEffect(() => {
-    const editors = contributors
-      .filter((contributor) => contributor.role === ContributorRole.EDITOR)
+    const filteredContributors = contributors
+      .filter((contributor) => contributor.role === contributorRole)
       .map((contributor, index) => ({ ...contributor, sequence: index + 1 }));
-    editorsRef.current = editors;
-    return () => {
-      editorsRef.current = [];
-    };
-  }, [contributors]);
+    contributorsRef.current = filteredContributors;
+  }, [contributors, contributorRole]);
```

Comment:
Mener epost-feltet var sykt laggy pga for mye rerendring en gang i tiden, så du kan jo bare dobbeltsjekke at det ikke får en ny smell nå her feks: http://localhost:3000/registration/dee38563-f88a-410f-8b15-1ce7432ee938?tab=2

Reply:
ser ut som det fungerer fint ja 😊 ingen lag og ingen feilmelding/warning